test/rdoc/test_rdoc_rubygems_hook.rb

```diff
@@ -2,10 +2,14 @@
 require 'rubygems'
 require 'fileutils'
 require 'tmpdir'
-require_relative '../../lib/rdoc/rubygems_hook'
 require 'test/unit'
 
-class TestRDocRubygemsHook < Test::Unit::TestCase
+begin
```

Comment:
This test requires `lib/rdoc/rubygems_plugin.rb` .

To execute this test under a ruby-core, `lib/rdoc/rubygems_plugin.rb` should be synced to `ruby/ruby`.
But I skip this test because of avoiding ambiguous about rubygems_plugins on `ruby/ruby`.